### ISSUE TITLE
Gnerate images with metric depth map

### ISSUE THREAD

**Issue by Jayzhang2333** [zkawfanx/Atlantis#18]
Hi there,

Awesome work! I am just wondering whether it is possible to genrated underwater images with metric depth maps (absolute depth map) instead of relative depth maps from MiDaS. I saw you used DIODE dataset to genertae underwater looking images, but DIODE dataset has dense ground truth metric depth map, did you still use MiDaS to generate depth? Thanks in advance!

**Comment by zkawfanx** [comment]
Hi, thanks for your interest in our work!

It is feasible to generate underwater images with metric depth maps. In our conference work, we tried to follow the training data distribution of ControlNet thus used MiDaS to estimate the relative depth of DIODE dataset for generation.

However, if you have a good enough ControlNet in hand (which has learned the features of underwater scenes and the correspondence of depth information and underwater textures), you can directly generate underwater images with metric depth after you turn it into disparity. For example, you can try to use the Flux.1-dev model and its official depth ControlNet for generation without additional training.

**Comment by Jayzhang2333** [comment]
Hi,

Thanks for the reply, really appreciate! I was thinking converting the dense metric depth into 8-bit disparity map similar to how MiDaS did with the SILog loss, but I will try with the ControlNet you mentioned.
